https://github.com/brailcom/speechd/commit/57e2c131f03ae8244b0478e449d86dea7d5724d8

musl additionally requires explicitly including sys/stat.h header for stat.
Including that would've been the proper patch if 'st' was actually used.

From 57e2c131f03ae8244b0478e449d86dea7d5724d8 Mon Sep 17 00:00:00 2001
From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Sun, 13 Mar 2022 18:35:59 +0100
Subject: [PATCH] Drop unused variable

---
 src/modules/espeak.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/modules/espeak.c b/src/modules/espeak.c
index 94a01197..1a47bb09 100644
--- a/src/modules/espeak.c
+++ b/src/modules/espeak.c
@@ -3,7 +3,7 @@
  * espeak.c - Speech Dispatcher backend for espeak
  *
  * Copyright (C) 2007 Brailcom, o.p.s.
- * Copyright (C) 2019-2021 Samuel Thibault <samuel.thibault@ens-lyon.org>
+ * Copyright (C) 2019-2022 Samuel Thibault <samuel.thibault@ens-lyon.org>
  *
  * This is free software; you can redistribute it and/or modify it
  * under the terms of the GNU General Public License as published by
@@ -880,7 +880,6 @@ static SPDVoice **espeak_list_synthesis_voices()
 		{
 			const char *identifier = espeak_mbrola[j]->identifier;
 			char *voicename, *dash, *path;
-			struct stat st;
 
 			totnummbrola++;
 
